### DISCUSSION TITLE
Technical Preview ended, can't find subscription option

### DISCUSSION THREAD

**Discussion by adellhk** [community/community#44151] [Copilot Conversations] [answered]
### Select Topic Area

Billing/Subscription management stymied

### Body

|1: preview access ended|2: Try to subscribe|3: Individual Plan Button -> goes to https://github.com/settings/billing|4: No options to subscribe in https://github.com/settings/billing/plans|5: also no billing management options in https://github.com/settings/copilot|
|-|-|-|-|-|
|![Screenshot 2023-01-12 at 1 08 30 PM](https://user-images.githubusercontent.com/9361067/212181601-be961e5d-1a3c-42fa-86a6-e38b0fe7e96e.png)|![Screenshot 2023-01-12 at 1 08 43 PM](https://user-images.githubusercontent.com/9361067/212181611-0ed6e020-9f43-46f1-a9b1-da42672a92aa.png)|![Screenshot 2023-01-12 at 1 08 35 PM](https://user-images.githubusercontent.com/9361067/212181604-0ba647d1-42b6-4ac3-a9d0-a67008a138e1.png)|![Screenshot 2023-01-12 at 1 08 41 PM](https://user-images.githubusercontent.com/9361067/212181607-d61545c8-072b-4d9b-ab0d-29898dc82779.png)|![Screenshot 2023-01-12 at 1 23 47 PM](https://user-images.githubusercontent.com/9361067/212184256-b1e49443-2023-4abc-b6ec-6dd23522c656.png)|

### Does anyone know how I can upgrade my subscription?

**Comment by iamcjwilliams** [comment, marked as answer]
We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
